app.R

```diff
@@ -34,14 +44,25 @@ ui <- fluidPage(
 
 # Define server logic required to draw a histogram
 server <- function(input, output) {
+    stem1 <- read_csv("data/scbi.stem1.csv") %>% na.omit()
+    stem2 <- read.csv("data/scbi.stem2.csv") %>% na.omit()
 
     output$distPlot <- renderPlot({
-        # generate bins based on input$bins from ui.R
-        x    <- faithful[, 2]
-        bins <- seq(min(x), max(x), length.out = input$bins + 1)
+        # generate bins based on input bin1 and bin2 from ui.R
+        bin1 <- seq(min(stem1$dbh), max(stem1$dbh), length.out = input$bin1 + 1)
+        bin2 <- seq(min(stem2$dbh), max(stem2$dbh), length.out = input$bin2 + 1)
 
         # draw the histogram with the specified number of bins
-        hist(x, breaks = bins, col = 'darkgray', border = 'white')
+        hist(stem1$dbh, breaks=bin1, ylim = c(0,30000),col=rgb(1,0,0,0.5), xlab="dbh",
```

Comment:
For smaller values of number of bins it seems `ylim = c(0,30000)` is not tall enough to capture the height of the highest bar. You might able to dynamically set base R histograms using the following code twice first, computing the joint max height, and then plotting as normal. Could you try to fix?

``` r
plot <- hist(faithful$eruptions, plot = FALSE)
plot$counts
#> [1] 55 37  5  9 34 75 54  3
max(plot$counts)
#> [1] 75
```

<sup>Created on 2020-09-28 by the [reprex package](https://reprex.tidyverse.org) (v0.3.0)</sup>

Reply:
Dear Prof. Kim
I changed the ylim and pushed the change from my forked master. I am not sure if I need to create a new pull request?

Reply:
yep! got it